<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="overview">
       <attribute name="title">
        <string>tr(Overview)</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QTableView" name="table"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item>
             <widget class="QPushButton" name="createButton">
              <property name="text">
               <string>tr(Create)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="text">
               <string>tr(Delete)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chart">
       <attribute name="title">
        <string>tr(Chart)</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menutr_Menu">
    <property name="title">
     <string>tr(Menu)</string>
    </property>
    <addaction name="actionGenerate"/>
   </widget>
   <addaction name="menutr_Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGenerate">
   <property name="text">
    <string>tr(Generate)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
